hedera-node/src/main/java/com/hedera/services/txns/ethereum/EthereumTransitionLogic.java

```diff
@@ -104,6 +115,9 @@ public void doStateTransition() {
 		if (syntheticTxBody.hasContractCall()) {
 			contractCallTransitionLogic.doStateTransitionOperation(syntheticTxBody, callingAccount.toId(), true);
 		} else if (syntheticTxBody.hasContractCreateInstance()) {
+			final var synthOp =
+					addInheritablePropertiesToContractCreate(syntheticTxBody.getContractCreateInstance(), callingAccount.toGrpcAccountId());
+			syntheticTxBody = TransactionBody.newBuilder().setContractCreateInstance(synthOp).build();
```

Comment:
Not sure it's braking anything rn, but probably we want to update the span map if we're updating the txBody, e.g. add:
			spanMapAccessor.setEthTxBodyMeta(txnCtx.accessor(), syntheticTxBody);